CMakeExternals/CTKData.cmake

```diff
@@ -36,7 +36,7 @@ if(NOT DEFINED CTKData_DIR)
     set(location_args GIT_REPOSITORY ${${proj}_GIT_REPOSITORY}
                       GIT_TAG ${revision_tag})
   else()
-    set(location_args GIT_REPOSITORY "${EP_GIT_PROTOCOL}://github.com/commontk/CTKData.git"
+    set(location_args GIT_REPOSITORY "https://github.com/commontk/CTKData.git"
```

Comment:
That commit should not be needed after integrating https://github.com/commontk/CTK/pull/1029

Reply:
We should hardcode GitHub anonymous git access URLs to always use https, and not a configurable protocol. By keeping `EP_GIT_PROTOCOL` in GitHub anonymous urls the code would become a bit misleading (as if we could use any other protocol than https), and may make it more difficult to use git protocol with other repositories (we could not set EP_GIT_PROTOCOL to git because that would prevent accessing GitHub repositories).

Reply:
Agreed. That makes sense. 👌

Reply:
> we should hardcode GitHub anonymous git access URLs to always use https, and not a configurable protocol

Here is a follow up pull request addressing this. See https://github.com/commontk/CTK/pull/1030